templates/16-04/index.html

```diff
@@ -1,7 +1,7 @@
 {% extends "16-04/base_16-04.html" %}
 
 {% block title %}Ubuntu 16.04 LTS (Xenial Xerus) {% endblock %}
-{% block meta_description %}Ubuntu 16.04 LTS (Xenial Xerus) enters the Extended Security Maintenance (ESM) period on 30 April 2021. ESM provides Ubuntu 16.04 LTS continued support and coverage beyond EOL with security patches for an additional three years.{% endblock %}
+{% block meta_description %}Ubuntu 16.04 LTS (Xenial Xerus) entered the Extended Security Maintenance (ESM) period on 30 April 2021. ESM provides Ubuntu 16.04 LTS continued support and coverage beyond EOL with security patches for an additional three years.{% endblock %}
```

Comment:
```suggestion
{% block meta_description %}Ubuntu 16.04 LTS (Xenial Xerus) entered the Extended Security Maintenance (ESM) period on 30 April 2021. ESM provides Ubuntu 16.04 LTS continued support and coverage beyond EOL with security patches for an additional five years.{% endblock %}
```

Reply:
Same thing here 
![image](https://user-images.githubusercontent.com/11927929/129007169-e8efcd80-d34e-4956-9c66-f8fdc19d1684.png)

Reply:
Got it! Didn't have the comments in view. Thanks for explaining 🙌